### PR TITLE
Use Roboto

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -10,7 +10,6 @@
   <link rel="stylesheet" href="{{site.baseurl}}/css/main.css">
   <link rel="stylesheet" href="{{site.baseurl}}/css/theme.css">
   <link rel="stylesheet" href="{{site.baseurl}}/css/pygments.css">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Google+Sans:400,500">
```
[the remark this answers is not in the document]
Comment (keanulee) [reply docs/_layouts/default.html]: No. I see it installed on my computer (Font Book), and I believe it's also included in Chrome.

**Comment by justinfagnani** [comment]
Are we still using Roboto over Open Sans?

And... does it matter if Chrome bundles a font if other browsers don't?

**Comment by justinfagnani** [comment]
Oh, I see the comment that lead to this... nvm.

File: docs/_layouts/default.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (justinfagnani) [reply docs/_layouts/default.html]: https://en.wikipedia.org/wiki/List_of_typefaces_included_with_macOS doesn't inlude Roboto

[the remark this answers is not in the document]
Comment (arthurevans) [reply docs/_layouts/default.html]: I was just looking at material.io and developer.google.com, which both appear to load roboto from fonts.googleapis.com. I guess worst case we fallback to sans-serif, just wondering why the different treatment for Roboto vs. Google Sans.

**Comment by hyperpress** [comment]
My understanding is Roboto was originally packaged by default for Polymer in keeping with Material Design, with a Sans backup. But I don't think it's necessary unless Google policy I'm not aware of.

**Pull request merged**